tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestPersonNameFormatter.java

```diff
@@ -429,7 +429,8 @@ public void TestExampleDependencies() {
         enWritable.add(namePath, "IRENE");
         exampleGenerator.updateCache(namePath);
 
-        String expected2 =  "〖Sinbad〗〖IRENE Adler〗〖John Hamish Watson〗〖Ada Cornelia Eva Sophia Meyer Wolf M.D. Ph.D.〗";
+        //String expected2 =  "〖Sinbad〗〖IRENE Adler〗〖John Hamish Watson〗〖Ada Cornelia Eva Sophia Meyer Wolf M.D. Ph.D.〗";
```

Comment:
Adding the allCaps IRENE did not work, we get the Titlecase one. Not sure why.

Reply:
Progress on this. The problem is that the ExampleGenerator path has hard-coded path dependencies. So I had to fix those.